xbmc/windowing/X11/GLContextEGL.cpp

```diff
@@ -174,8 +176,7 @@ bool CGLContextEGL::Refresh(bool force, int screen, Window glWindow, bool &newCo
       return false;
     }
 
-    CLog::Log(LOGWARNING, "Failed to get an OpenGL context supporting core profile 3.2,  \
-                             using legacy mode with reduced feature set");
+    CLog::Log(LOGWARNING, "Failed to get an OpenGL context supporting core profile 3.2, using legacy mode with reduced feature set");
```

Comment:
Why make this a long line?

Reply:
ok, making it a long line wasn't the correct solution. instead I should have deleted the spaces/tabs in the second line, because the actual log output looked very ugly.